helm/templates/controller/statefulset.yaml

```diff
@@ -55,6 +52,12 @@ spec:
             requests:
               cpu: 10m
               memory: 64Mi
+          volumeMounts:
+            - name: nats-dats
+              mountPath: /tmp
```

Comment:
I think the name of the volume has a typo, plus so not set that to be `/tmp`, make it something specific to NATS please